docs/content/src/learn/tutorials/messaging-integrations/messaging-with-kafka/README.md

```diff
@@ -0,0 +1,151 @@
+# Messaging with Kakfa
+
+Kafka is a distributed, partitioned, replicated commit log service. It provides the functionality of a publish-subscribe messaging system with a unique design. Kafka mainly operates based on a topic model. A topic is a category or feed name to which records get published. Topics in Kafka are always multi-subscriber.
+
+This guide walks you through the process of messaging with Apache Kafka using Ballerina language. The following figure illustrates the publish-subscribe messaging use-case. 
+
+![messaging_with_kafka](/resources/diagram.png "Messaging with Kafka")
```

Comment:
Add this under the "What you'll build" section. Also, you need to host the image in this location: https://github.com/wso2/ballerina-integrator/tree/master/docs/content/src/assets/img

Reply:
Done